src/app/components/timeline/LineTitleCell.js

```diff
@@ -394,7 +419,7 @@ function mapStateToProps(state, ownProps) {
 
 function mapDispatchToProps(dispatch, ownProps) {
   return {
-    actions: bindActionCreators(LineActions, dispatch),
+    actions: bindActionCreators(Object.assign({}, LineActions, uiActions), dispatch),
```

Comment:
What we normally do here is add another line below this, like so:

```
uiActions: bindActionCreators(uiActions, dispatch),
```

Reply:
done!